program/rust/src/processor/upd_price.rs

```diff
@@ -197,7 +183,6 @@ pub fn upd_price(
     let mut price_data = load_checked::<PriceAccount>(price_account, cmd_args.header.version)?;
 
     // Feature-gated accumulator-specific code, used only on pythnet/pythtest
-    #[cfg(feature = "pythnet")]
```

Comment:
I think a good refactor you can already to here is move the block :
`        if aggregate_updated {
            price_data.message_sent_ = 0;
            price_data.update_price_cumulative()?;
        }`
closer to the `c_upd_twap` code